Collections File Level Annotations/mPowerRerunFileAnnotations.R

```diff
@@ -0,0 +1,869 @@
+########################################################
+#' This file is used for annotating mPowerRerun project
+#' with the required annotations used for the
+#' the dHealth portal
+#' @author: aryton.tediarjo@sagebase.org
+########################################################
+
+###################
+# Required libraries
+####################
+library(plyr)
+library(tidyverse)
+library(synapser)
+library(glue)
+
+synLogin()
+
+############################
+# Global Variables
+############################
+STUDY <- 'mPower - A Smartphone Approach to Remotely Monitor PD and Individual Response to Therapy'
+URL <- URLencode(STUDY,  reserved = FALSE, repeated = FALSE)
+PROJECT_ID <- 'syn23277418'
+FILE_VIEW_ID <- "syn23545224"
+PROJECT_LINK <- glue::glue("[{STUDY}](/Explore/Studies/DetailsPage?study={URL})")
+INDEX <- FALSE
+
+############################
+# Tables for dHealth
+############################
+
+#' all resource used for the mPower rerun project
+#' which will index all data used for analysis
+#' both for the 6 months and the public data
+tables.annotations <- list(
+    "syn23545224" = list(consortium = 'mHealth',
+                         study = STUDY,
+                         studyOrProject = PROJECT_LINK,
+                         dataCollectionMethod= c('active'),
+                         deviceType = c('handheld'),
+                         sensorType = c('accelerometer', 'gyroscope', 'magnetometer', "microphone"),
+                         devicePlatform = c('iOS'), 
+                         deviceLocation = c('hand', 'pocket', "flat surface"),
+                         diagnosis= c("Parkinson's disease", 'control'),
+                         reportedOutcome = c('medication report'),
+                         digitalAssessmentCategory= c('gait', 
+                                                      'phonation',
+                                                      'motor coordination'),
+                         digitalAssessmentDetails = c('walking (flat)', 
+                                                      'standing', 
+                                                      'phonation',
+                                                      'motor coordination'),
```

Comment:
remove 'motor coordination' from digitalAssessmentDetails, we already have it in digitalAssessmentCategory.

Check this for all digitalAssessmentDetails annotations in this file.